mmv1/third_party/terraform/website/docs/r/compute_shared_vpc_service_project.html.markdown

```diff
@@ -18,6 +18,8 @@ For more information, see,
 [the Project API documentation](https://cloud.google.com/compute/docs/reference/latest/projects),
 where the Shared VPC feature is referred to by its former name "XPN".
 
+~> **Note:** If Shared VPC Admin role is set at the folder level, use the google-beta provider. The google provider only supports this permission at project or organizational level currently. [[0]](https://cloud.google.com/vpc/docs/provisioning-shared-vpc#enable-shared-vpc-host)
```

Comment:
Is Org level confirmed to work?

Reply:
Org level is notated in the [official documentation](https://cloud.google.com/vpc/docs/provisioning-shared-vpc#enable-shared-vpc-host), project from the bug and client.